.github/workflows/codeql.yml

```
@@ -87,6 +87,6 @@ jobs:
           exit 1
 
       - name: Perform CodeQL Analysis
-        uses: github/codeql-action/analyze@64d10c13136e1c5bce3e5fbde8d4906eeaafc885 # v3
+        uses: github/codeql-action/analyze@a8d1ac45b9a34d11fe398d5503176af0d06b303e # v3
```

```suggestion
        uses: github/codeql-action/init@a8d1ac45b9a34d11fe398d5503176af0d06b303e # v3.30.7
```

looked up this version from https://github.com/github/codeql-action/releases – https://github.com/github/codeql-action/releases/tag/v3.30.7 is the tag on https://github.com/github/codeql-action/commit/a8d1ac45b9a34d11fe398d5503176af0d06b303e